src/context/AuthContext.tsx

```diff
@@ -0,0 +1,36 @@
+import { createContext, ReactNode, useEffect, useState } from 'react'
+import { GetToken } from '../components/auth/security'
+
+interface IAuthContext {
+  isAuth: boolean
+  login: (token: string) => void
+  logout: () => void
+}
+
+export const AuthContext = createContext({} as IAuthContext)
+
+export const AuthProvider = ({ children }: { children: ReactNode }) => {
+  const token = GetToken()
+
+  const [isAuth, setIsAuth] = useState(token ? true : false)
+
+  useEffect(() => {
+    token ? setIsAuth(true) : localStorage.removeItem('login')
```

Comment:
It should setIsAuth to false if doesn't have token too no?

Reply:
yeah, I forgot about it.